sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -230,9 +236,14 @@ export default defineComponent({
           data: applicationWizard.submission.data,
           associatedFiles,
         });
-        snackBar.success("Application draft saved with success.");
+        savedDraftData = structuredClone(applicationWizard.submission.data);
```

Comment:
When I was talking to @andrewsignori-aot he suggested a good idea to define `savedDraftData` as a string and always assign like this. This will eliminate the need for using `structuredClone`.
```ts
savedDraftData = JSON.stringify(applicationWizard.submission.data);
```